src/rust/ensogl/src/math/geometry/circle.rs

```diff
@@ -0,0 +1,3 @@
+//! Root module for circle-related functionality.
+
+pub mod circle_segment;
```

Comment:
I do not like this naming tbh as we are repeating ourselves `circle::circle_segment`. it should be `circle::segment` instead.